### PR TITLE
feat: change admin api key auth to merchant api key auth in few connectors flow

### PR DESCRIPTION
**Pull request by hrithikesh026**
## Type of Change

- [ ] Bugfix
- [x] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Description

Currently below apis use either `jwt` auth or `admin api key` auth. I changed `admin api key` auth to `merchant api key` for these apis.
1. connector retrieve
2. connector update
3. connector list

Also made respective changes to cypress tests and postman collections.

### Additional Changes

- [ ] This PR modifies the API contract
- [ ] This PR modifies the database schema
- [ ] This PR modifies application configuration/environment variables

## How did you test it?

Ran cypress tests in my branch as well as main. Same tests were failing in both branches.
image

ran stripe postman test.
image

## Checklist

- [x] I formatted the code `cargo +nightly fmt --all`
- [x] I addressed lints thrown by `cargo clippy`
- [x] I reviewed the submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

File: cypress-tests/cypress/support/commands.js
Comment (pixincreate) [inline cypress-tests/cypress/support/commands.js:1]: Can you please run tests once and update the PR description?

File: postman/collection-json/stripe.postman_collection.json
Comment (pixincreate) [inline postman/collection-json/stripe.postman_collection.json:1]: Please run tests against this and update the PR description.

**Review by pixincreate — COMMENTED** [review]
Code wise, looks good.

File: cypress-tests/cypress/support/commands.js
Comment (hrithikesh026) [reply cypress-tests/cypress/support/commands.js:1]: I've already attached the screenshots

File: postman/collection-json/stripe.postman_collection.json
Comment (hrithikesh026) [reply postman/collection-json/stripe.postman_collection.json:1]: I've already attached the screenshots

**Review by jarnura — APPROVED** [review]

**Review by pixincreate — APPROVED** [review]

**Review by ThisIsMani — APPROVED** [review]

**Pull request merged**